src/clockkit.cs

```diff
@@ -62,9 +62,13 @@ interface CLKComplicationDataSource {
 		[Export ("requestedUpdateBudgetExhausted")]
 		void RequestedUpdateBudgetExhausted ();
 
-		[Abstract]
+		// this was @required in watchOS 2.x but is now deprecated and downgraded to @optional in watchOS 3 (betas)
```

Comment:
@rolfbjarne I'm still undecided if we should backport the master/C8 to pre-empt the change. Any thoughts ?